userspace/libsinsp/parsers.cpp

```diff
@@ -978,7 +978,6 @@ void sinsp_parser::parse_clone_exit_caller(sinsp_evt *evt, int64_t child_tid)
 	if(caller_tinfo == nullptr)
 	{
 		/* Invalidate the thread info associated with this event */
-		ASSERT(false);
```

Comment:
For the most part we don't have these and other ASSERT statements. Therefore I considered them legacy code and removed them in order to enforce a more consistent style.

Reply:
Thank you. We need to get rid of legacy asserts!